### PR TITLE
Delete the timer based measurement since we've switched over to

### PR DESCRIPTION
**Pull request by rniwa**
requestAnimationFrame based measurement for 3.0 and found no issues.

### CODE REVIEW

File: resources/params.mjs [hunk 1]
```diff
@@ -122,8 +122,8 @@ class Params {
         if (!searchParams.has("measurementMethod"))
             return defaultParams.measurementMethod;
         const measurementMethod = searchParams.get("measurementMethod");
-        if (measurementMethod !== "timer" && measurementMethod !== "raf")
-            throw new Error(`Invalid measurement method: '${measurementMethod}', must be either 'raf' or 'timer'.`);
+        if (measurementMethod !== "raf")
```
Comment (rniwa) [inline resources/params.mjs:125]: I kept this here since we may want to support "async" as an option now.

File: resources/shared/params.mjs [hunk 2]
```diff
@@ -122,8 +122,8 @@ export class Params {
         if (!searchParams.has("measurementMethod"))
             return defaultParams.measurementMethod;
         const measurementMethod = searchParams.get("measurementMethod");
-        if (measurementMethod !== "timer" && measurementMethod !== "raf")
-            throw new Error(`Invalid measurement method: '${measurementMethod}', must be either 'raf' or 'timer'.`);
+        if (measurementMethod !== "raf")
+            throw new Error(`Invalid measurement method: '${measurementMethod}', must be either 'raf'.`);
```
Comment (flashdesignory) [inline resources/shared/params.mjs:126]: should we change it to:
```
throw new Error(`Invalid measurement method: '${measurementMethod}', must be 'raf'.`);
```

**Review by flashdesignory — APPROVED** [review]

File: resources/shared/params.mjs (hunk 2 above)
Comment (smaug----) [reply resources/shared/params.mjs:126]: yeah, drop "either"

**Review by smaug---- — APPROVED** [review]

**Review by julienw — APPROVED** [review]

**Comment by camillobruni** [comment]
- Removing the timer-based code sounds good
- Can we keep the UI so we can switch between the current rAF and the new async runner?

**Comment by rniwa** [comment]
> * Can we keep the UI so we can switch between the current rAF and the new async runner?

I don't know how keeping the UI mean here because it's a checkbox with "rAF timing" label. Are you thinking of replacing that with "Async timing" or something?

**Pull request merged**